style.css

```diff
@@ -0,0 +1,70 @@
+body {
+  width: 320px;
+  background-color: #e5e5e5;
+  margin: 50px;
+}
+
```

Comment:
- [ ] Kindly remove `width: 320px;` role, because it will work fine just on this but with any other width it will be broken.
- [ ] Kindly change the `margin: 50px;` to `margin: 0;` to remove this not nesseccary margin.

![image](https://user-images.githubusercontent.com/35535130/158730787-266583c2-0a2d-483f-bf57-eb5dc46fd1b3.png)

Reply:
resolved to a certain extend